Xamarin.Forms.Platform.UAP/LabelRenderer.cs

```diff
@@ -282,22 +282,30 @@ void UpdateLineBreakMode(TextBlock textBlock)
 				case LineBreakMode.HeadTruncation:
 					// TODO: This truncates at the end.
 					textBlock.TextTrimming = TextTrimming.WordEllipsis;
-					textBlock.TextWrapping = TextWrapping.NoWrap;
+					DetermineTextWrapping(textBlock);
 					break;
 				case LineBreakMode.TailTruncation:
 					textBlock.TextTrimming = TextTrimming.CharacterEllipsis;
-					textBlock.TextWrapping = TextWrapping.NoWrap;
+					DetermineTextWrapping(textBlock);
 					break;
 				case LineBreakMode.MiddleTruncation:
 					// TODO: This truncates at the end.
 					textBlock.TextTrimming = TextTrimming.WordEllipsis;
-					textBlock.TextWrapping = TextWrapping.NoWrap;
+					DetermineTextWrapping(textBlock);
 					break;
 				default:
 					throw new ArgumentOutOfRangeException();
 			}
 		}
 
+		void DetermineTextWrapping(TextBlock textBlock)
```

Comment:
```suggestion
		void DetermineTruncatedTextWrapping(TextBlock textBlock)
```

Reply:
It is necessary to change the names of this method in calls...

Reply:
Haha yeah. I thought that to myself right after accepting the suggested change... I'll do it now